blog.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Blog</title>
+    <link rel="stylesheet" href="./styles/master.css">
+    <style>
+      body {
+         background-color: lightblue;
+      }
+    </style>
+  </head>
+
+  <body>
+    <header>
+      <img src="images/adoptadog-logo.svg" width="200" alt="adoptadog logo">
+      <p>Seattle's premiere dog adoption agency</p>
+
+      <img src="images/cart-icon.svg" width="24" alt="shopping cart icon">
+      <p>$1234.56</p>
+      <button type="button">Checkout</button>
+
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="dogs.html">Dogs</a></li>
+          <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+    </header>
+
+
+    <main>
+      <h2>Adoptadog Blog</h2>
+        <section class="blog1">
```

Comment:
Since all of these blogs are styled the same, you could use the same class for each blog section. Also, consider another HTML5 element for these blog _articles_